spec.go

```diff
@@ -438,6 +438,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
 	if err != nil {
 		return nil, err
 	}
+	if spec.Linux.CgroupsPath != "" {
+		name = spec.Linux.CgroupsPath
+	}
```

Comment:
We should remove parent as you and @lk4d4 discussed. CgroupsPath must not be a full path, but relative to cgroup mount on host. That's what I interpret from spec.